eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Gateway/DoctrineDatabase.php

```diff
@@ -1250,14 +1250,14 @@ private function loadLocationEntriesMatchingMultipleLanguages($locationId, array
             ->from($this->table)
             ->where('action = :action')
             // fetch rows matching any of the given Languages
-            ->where('lang_mask & :languageMask <> 0')
+            ->andWhere('lang_mask & :languageMask <> 0')
             ->setParameter(':action', 'eznode:' . $locationId)
             ->setParameter(':languageMask', $languageMask)
         ;
 
         $statement = $query->execute();
-        while (false !== ($row = $statement->fetch(\PDO::FETCH_ASSOC))) {
-            yield $row;
```

Comment:
I've discovered another issue here. Travis CI was failing because SQLite driver created here an endless loop yielding results that were created in another step of that operation instead of returning result set which was obtained during `execute()` call. 

It seems driver-specific, so for normal installation on MySQL it works. I'm not 100% that's the reason, just a wild guess. Replacing Generator with an array worked. cc @andrerom @adamwojs